src/main/java/org/apache/commons/io/output/FileWriterWithEncoding.java

```diff
@@ -245,7 +240,7 @@ public FileWriterWithEncoding(final String fileName, final String charsetName, f
      */
```

Comment:
@wodencafe do we need to override the methods below? Could we just use the parent methods directly?

Reply:
Good point, we should just let the super class handle it.